applications/dashboard/models/class.usermodel.php

```diff
@@ -1051,13 +1051,15 @@ public function joinUsers(&$data, $columns, $options = []) {
         }
 
         // Join the user data using prefixes (ex: 'Name' for 'InsertUserID' becomes 'InsertName')
```

Comment:
I have high confidence in this change not breaking anything because there are is_object checks around each access to the rows and fallbacks in that event.